review/rhea-promise.api.md

```diff
@@ -0,0 +1,589 @@
+    MovedPermanently = 301,
+    // (undocumented)
+    MultipleChoices = 300,
+    // (undocumented)
+    NoContent = 204,
+    // (undocumented)
+    NonAuthoritativeInformation = 203,
+    // (undocumented)
+    NotAcceptable = 406,
+    // (undocumented)
+    NotFound = 404,
+    // (undocumented)
+    NotImplemented = 501,
+    // (undocumented)
+    NotModified = 304,
+    // (undocumented)
+    OK = 200,
+    // (undocumented)
+    PartialContent = 206,
+    // (undocumented)
+    PaymentRequired = 402,
+    // (undocumented)
+    PreconditionFailed = 412,
+    // (undocumented)
+    ProxyAuthenticationRequired = 407,
+    // (undocumented)
+    Redirect = 302,
+    // (undocumented)
+    RedirectKeepVerb = 307,
+    // (undocumented)
+    RedirectMethod = 303,
+    // (undocumented)
+    RequestedRangeNotSatisfiable = 416,
+    // (undocumented)
+    RequestEntityTooLarge = 413,
+    // (undocumented)
+    RequestTimeout = 408,
+    // (undocumented)
+    RequestUriTooLong = 414,
+    // (undocumented)
+    ResetContent = 205,
+    // (undocumented)
+    SeeOther = 303,
+    // (undocumented)
+    ServiceUnavailable = 503,
+    // (undocumented)
+    SwitchingProtocols = 101,
+    // (undocumented)
+    TemporaryRedirect = 307,
+    // (undocumented)
+    Unauthorized = 401,
+    // (undocumented)
+    UnsupportedMediaType = 415,
+    // (undocumented)
+    Unused = 306,
+    // (undocumented)
+    UpgradeRequired = 426,
+    // (undocumented)
+    UseProxy = 305
+}
+
+// @public
+export interface AwaitableSender {
+    // (undocumented)
+    on(event: SenderEvents, listener: OnAmqpEvent): this;
+}
+
+// Warning: (ae-forgotten-export) The symbol "BaseSender" needs to be exported by the entry point index.d.ts
+//
+// @public
+export class AwaitableSender extends BaseSender {
+    constructor(session: Session, sender: Sender_2, options?: AwaitableSenderOptions);
+    deliveryDispositionMap: Map<number, PromiseLike_2>;
+    send(msg: Message | Buffer, options?: AwaitableSendOptions): Promise<Delivery>;
+}
+
+// Warning: (ae-forgotten-export) The symbol "BaseSenderOptions" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type AwaitableSenderOptions = BaseSenderOptions;
+
+// @public (undocumented)
+export interface AwaitableSendOptions {
+    // Warning: (ae-forgotten-export) The symbol "AbortSignalLike" needs to be exported by the entry point index.d.ts
+    abortSignal?: AbortSignalLike;
+    format?: number;
+    tag?: Buffer | string;
+    timeoutInSeconds?: number;
+}
+
+// @public
+export interface Connection {
+    // (undocumented)
+    on(event: ConnectionEvents, listener: OnAmqpEvent): this;
+}
+
+// Warning: (ae-forgotten-export) The symbol "Entity" needs to be exported by the entry point index.d.ts
+//
+// @public
+export class Connection extends Entity {
+    // Warning: (ae-forgotten-export) The symbol "CreatedRheaConnectionOptions" needs to be exported by the entry point index.d.ts
```

Comment:
Thanks for this PR. I think not having an api document caused all these missed exports.